tests/elements-regression/tests/templates/text_editor.json

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
You are missing some of the permutations that we had in the original tests
Also, there are missing templates for `image` and `button`

Reply:
fixed. Also added some description of what is being tested in templates.